File: csharpHomeWork/csharpHomeWork/FFNaghtly.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium;
using NUnit.Framework;

namespace csharpHomeWork
{
    [TestClass]
    public class FFNaghtly
    {
        private IWebDriver driver;

        [SetUp]
        public void set()
        {
            
            FirefoxOptions opt = new FirefoxOptions();
            opt.BrowserExecutableLocation = @"D:\Mozilla_Firefox_Nightly\firefox.exe";
            this.driver = new FirefoxDriver(opt);
        }
        [Test]
        public void FFNaghtlyTesT()
        {
           
            this.driver.Url = "http://localhost:81/lifecart/admin/";

            this.driver.FindElement(By.Name("username")).Clear();
            this.driver.FindElement(By.Name("username")).SendKeys("admin");
            this.driver.FindElement(By.Name("password")).Clear();
            this.driver.FindElement(By.Name("password")).SendKeys("admin");
            this.driver.FindElement(By.Name("login")).Click();

        }
        [TearDown]
        public void close()
        {
            this.driver.Quit();
            this.driver = null;
        }
    }
}

File: untitled/javaHomeWork/src/test/java/FFNaghtly.java
import org.junit.After;
import org.junit.Before;
import org.junit.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxBinary;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.firefox.FirefoxProfile;
import org.openqa.selenium.remote.DesiredCapabilities;

import java.io.File;

/**
 * Created by boltenkov on 17.03.2017.
 */
public class FFNaghtly {
    private WebDriver driver;


    @Before
    public void start()
    {

        DesiredCapabilities caps = new DesiredCapabilities();
        caps.setCapability(FirefoxDriver.MARIONETTE,true);
        caps.setCapability("unexpectedAlertBehaviour","ignore");
        this.driver= new FirefoxDriver(new FirefoxBinary(new File("D:\\Mozilla_Firefox_Nightly\\firefox.exe")),new FirefoxProfile(), caps);
    }

    @Test
    public void myFirst()
    {
        this.driver.get("http://localhost:81/lifecart/admin/");
        this.driver.findElement(By.name("username")).clear();
        this.driver.findElement(By.name("username")).sendKeys("admin");
        this.driver.findElement(By.name("password")).clear();
        this.driver.findElement(By.name("password")).sendKeys("admin");
        this.driver.findElement(By.name("login")).click();


    }

    @After
    public void stop()
    {
        this.driver.quit();
        this.driver = null;
    }
}
